get_deps.py

```diff
@@ -30,11 +37,22 @@ def main():
 		if dll_name.startswith('???'):
 			print('Unknown DLL?')
 			continue
-		if dll_path.lower().startswith('/c/windows') or dll_path.lower().startswith('c:/windows'):
+		# ldd on msys gives Unix-style paths, but mingw Python wants them Windows-style
+		# Use cygpath to convert the paths, because both mingw and msys Python can handle them
+		dll_path = subprocess.check_output(['cygpath', '-w', dll_path]).strip()

```

Comment:
nit: `-m` (I always prefer forward slashes)

*Would also need change in comparison below and might have other implications; again this is nitpicking = optional request.*